Source/WebInspectorUI/UserInterface/Controllers/JavaScriptRuntimeCompletionProvider.js

```diff
@@ -332,7 +346,6 @@ WI.JavaScriptRuntimeCompletionProvider = class JavaScriptRuntimeCompletionProvid
                     break;
                 }
 
-                // FIXME: Due to caching, sometimes old $n values show up as completion results even though they are not available. We should clear that proactively.
                 for (var i = 1; i <= WI.ConsoleCommandResultMessage.maximumSavedResultIndex; ++i) {
```

Comment:
Would the code be a bit prettier if we avoid reading the global-ish variable `maximumSavedResultIndex` from here, but instead let the variable be a instance variable here and let `ConsoleCommandResultMessage` reach into this `WI.javaScriptRuntimeCompletionProvider` to modify it instead?

If you think that'd be an improvement, I can make a bug report to do that next.

Otherwise, my way of reaching into this completion provider object to clear its cache feels a little inconsistent to what this is doing (reaching elsewhere).

Reply:
are we sure that this FIXME is no longer valid?  i can imagine a number of scenarios where it's still possible to have invalid cached property names

Reply:
If the FIXME has been tested in the mini-browser with either the code changes or in general, we can delete the FIXME. If not, we should probably keep it.